ctapipe/image/toymodel.py

```diff
@@ -369,3 +371,72 @@ def pdf(self, x, y):
         r = np.sqrt((x - self.x) ** 2 + (y - self.y) ** 2)
 
         return norm(self.radius.to_value(u.m), self.sigma.to_value(u.m)).pdf(r)
+
+
+class SkewedCauchy(ImageModel):
```

Comment:
It's not a `SkewedCauchy`, is it? It's a SkewedGaussian for the longitudinal and a normal cauchy for trans, right?

Reply:
Yes. I will clarify